### ISSUE TITLE
Firefox websocket upgrade request

### ISSUE THREAD

**Issue by smfgh** [robaho/httpserver#12]
Thank for your websocket-implementation. :-)

In WebSocketHandler.isWebsocketRequest there is a check for the connection upgrade, which is working well for Chrome and Opera, but my Firefox (version 133.0 snap) fails to connect due to sending `keep-alive, Upgrade` instead of the expected `Upgrade`.

Changing the first if-clause in isWebsocketRequest from
```
  if (connection == null ||
    !connection.equalsIgnoreCase(Util.HEADER_CONNECTION_VALUE)) {

```
to something like
 ```
 if (connection == null ||
     !connection.toLowerCase()
        .contains(Util.HEADER_CONNECTION_VALUE.toLowerCase())) {
```
fixes this issue for me.

**Comment by robaho** [comment]
Hmm, the code does look wrong, but there is a test case that verifies this [here](https://github.com/robaho/httpserver/blob/85530fa455623b0596814b017f945fc465fe4f37/src/test/java/robaho/net/httpserver/websockets/WebSocketResponseHandlerTest.java#L155-L157) so I must have altered the code when integrating nanohttp code.

More importantly, it brings up an issue, because according to the spec, it should be a comma delimited list, but I believe this syntax is also correct:

```
Connection: keep-alive
Connection: Upgrade
```

so using getFirst() is not appropriate either.

**Comment by robaho** [comment]
Ah, the test is broken, it isn't verifying the results!

**Comment by robaho** [comment]
This has been fixed in version 1.0.11

**Comment by smfgh** [comment]
thank you so much! :-)
